src/app/components/job/details/job-configuration.html

```diff
@@ -38,8 +38,8 @@
                 <span *ngIf="job?.executionInfo?.endTime">Completed in: {{ executionInfo.runtime }}</span>
             </div>
 
-            <bl-text-property label="Start time" [value]="executionInfo.startTime"></bl-text-property>
-            <bl-text-property label="End time" [value]="executionInfo.endTime"></bl-text-property>
+            <bl-date-property label="Start time" [value]="job?.executionInfo.startTime"></bl-date-property>
```

Comment:
I also don't think you need the `?` after job. This component assume a job is being passed or it should most likely do a check at the top

Reply:
there is a lot of other uses in this file which dont have it so it would crash on those anyway

Reply:
All good I'll change